docs/source/conf.py

```diff
@@ -52,7 +52,7 @@ def __getattr__(cls, name):
                 'libtorrent', 'psyco', 'rencode',
                 'twisted.web', 'twisted.web.client', 'twisted.web.error',
                 'win32file', 'win32event', 'win32gui', 'win32api', 'win32con', '_winreg',
-                'pygtk', "gtk", "gobject", "gtk.gdk", "pango", "cairo", "pangocairo", "chardet"]
+                'pygtk', "gtk", "gobject", "Gtk.gdk", "pango", "cairo", "pangocairo", "chardet"]  # FIXME
```

Comment:
You have to leave more information than just FIXME, also don't put comments at the end of the line... https://www.python.org/dev/peps/pep-0350/#examples